template_README.md

```diff
@@ -28,43 +14,46 @@ Below are instructions for how to deploy this NOMAD Oasis distribution
 ### For a new Oasis
 
 1. Make sure you have [docker](https://docs.docker.com/engine/install/) installed.
-Docker nowadays comes with `docker compose` built in. Prior, you needed to
-install the stand-alone [docker-compose](https://docs.docker.com/compose/install/).
-2. Get the `nomad-oasis.zip` archive from your distribution repository using for example curl
-```sh
-curl -L -o nomad-oasis.zip "https://github.com/GITHUB_REPOSITORY/raw/main/nomad-oasis.zip"
-```
-3. Unzip the `nomad-oasis.zip` file and enter the extracted directory
-```sh
-unzip nomad-oasis.zip
-cd nomad-oasis
-```
-4. _On Linux only,_ recursively change the owner of the `.volumes` directory to the nomad user (1000) 
+   Docker nowadays comes with `docker compose` built in. Prior, you needed to
+   install the stand-alone [docker-compose](https://docs.docker.com/compose/install/).
+2. _On Linux only,_ recursively change the owner of the `.volumes` directory to the nomad user (1000)
+
 ```sh
 sudo chown -R 1000 .volumes
 ```
-5. Pull the images specified in the `docker-compose.yaml` (retrieved from the `nomad-oasis.zip`).
+
+5. Pull the images specified in the `docker-compose.yaml`
```

Comment:
Step 3-4? I think we should still have a compressed archive that the users can just curl.

Reply:
The docker compose file and the configs are part of the repo now, why'd an curl step be necessary?